index.html

```diff
@@ -190,5 +190,40 @@ <h2 class="aboutmeintro">About me</h2>
       </div>
 
     </section>
+    <footer>
+      <div class="footer">
+      <form class="form"action="https://formspree.io/f/mjvzwzlb" method="post">
+        <h2 class="footer-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <ul class="ul-form">
+          <li><input maxlength="30"
+            type="name"
+            placeholder="Full name" class="form-name"
+            required></li>
+          <li><input type="email"
+            name="email"
+            id="email"
+            class="form-Email"
+            placeholder="Email address"
+            required></li>
+          <li class="textarea-box"><textarea
+            type="textarea"
+             id="message"
+            class="input-textarea"
+            name="message"
+            maxlength="500"
+            placeholder="Enter text"
+            required></textarea></li>
+        </ul>
+        <button class="submit-form"type="submit"> Get it touch</button>
+      </div>
+      </form>
```

Comment:
Kindly move your contact-form out of the footer section since this is not a part of the footer of webpage 🤗

Reply:
hello sir but here in the website requirements it says inside the footer section and I quote "the contact form in the footer"
![Screenshot (8)](https://user-images.githubusercontent.com/103601335/192563760-1b64d410-dab0-456a-a7a2-e71102623b1f.png)